pages/learn/foundations/how-nextjs-works/cdns-and-edge.mdx

```diff
@@ -16,4 +18,58 @@ import NotTranslateYet from '@components/NotTranslateYet';
   ]}
 />
 
-<NotTranslateYet/>
+## 網路是什麼？
+
+了解你的應用程式部屬到網路上時，程式碼存放在何處及從哪裡運行是非常有幫助的。
+你可以把網路想像成一群彼此連結的電腦（或是伺服器），而它們之間能夠共享資源。
+就 Next.js 應用程式而言，你的程式碼能夠被分送到 **原始伺服器**，**內容分發網路（CDNs）** 以及 **邊緣運算** 位置。
+讓我們看看這些名詞各自代表什麼：
+
+### 原始伺服器（Origin Servers）
+
+如我們剛剛討論的，這個伺服器指的是一台用來儲存及運行原始版本的應用程式程式碼的電腦。
+
+我們使用 **原始（origin）** 來和其他被分配資源的伺服器（像是 **CDN 伺服器** 及 **邊緣運算伺服器**）進行概念上的區分。
+
+當原始伺服器接收請求後，在進行回應之前會做一些運算工作。運算過後的結果可以被轉移到 CDN（內容分發網路） 存放。
+
+
+### 內容分發網路（Content Delivery Networks）
+
+CDNs（Content Delivery Networks 縮寫）將靜態資源（像是HTML 及 圖片檔）儲存在世界多個地方，且位置存在於客戶端與原始伺服器之間。當客戶端發起新的請求，離使用者最近的 CDN 伺服器會回應已事先進行快取的靜態資源。
```

Comment:
```suggestion
CDNs（Content Delivery Networks 縮寫）將靜態資源（像是 HTML 及圖片檔）儲存在世界多個地方，且位置存在於客戶端與原始伺服器之間。當客戶端發起新的請求，離使用者最近的 CDN 伺服器會回應已事先進行快取的靜態資源。
```

Reply:
Problems all solved. Please check it out！